nilearn/connectome/tests/test_connectivity_matrices.py

```diff
@@ -43,12 +43,6 @@
 N_SUBJECTS = 5
 
 
-@parametrize_with_checks(estimators=[EmpiricalCovariance(), LedoitWolf()])
```

Comment:
removing because EmpiricalCovariance and LedoitWolf actually come from sklearns, so no need to check their compliance with sklearn estimators